tests/test_models/test_catboost.py

```diff
@@ -144,3 +144,71 @@ def test_save_load(model, example_tsds):
     horizon = 3
     transforms = [LagTransform(in_column="target", lags=list(range(horizon, horizon + 3)))]
     assert_model_equals_loaded_original(model=model, ts=example_tsds, transforms=transforms, horizon=horizon)
+
+
+@pytest.fixture
+def ts_w_exog(simple_df_relevance):
```

Comment:
We actually can use the old version as now you also need to create the new fixture